### PR TITLE
README.md: fix for broken tutorial command-line, to stop picking up t…

### PR DESCRIPTION
**Pull request by kenmccracken**
…he -sources jar file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,26 +136,26 @@ The examples directory contains a tutorial example.  To run it
...
+     $ java -jar examples/here-oauth-client-example/target/here-oauth-client-example-*[!javadoc][!sources].jar -v
 
 to print a full valid HERE Access Token to stdout. You can also put the file in a different
 location or give it a different name, just supply the file as input to the executable jar command
 line.  The examples are for tutorial purposes only and MUST NOT be used in your deployed
 application.  You might find it useful to start from the main(..) method's sample code, and
 adapt the integration to your environment.
 
-You can use  the -idToken option to output the id token(Open  Id) instead of the Here Access Token.
+You can use  the -idToken option to output the HERE Id Token (in Open ID format) instead of the
```
Comment (sschuberth) [inline README.md]: Nit: Could you please only use a single space between "use" and "the"? Also, apply formatting with backticks like `-idToken`?

File: README.md [hunk 2]
```diff
@@ -136,26 +136,26 @@ The examples directory contains a tutorial example.  To run it
...
-You can use  the -idToken option to output the id token(Open  Id) instead of the Here Access Token.
+You can use  the -idToken option to output the HERE Id Token (in Open ID format) instead of the 
+HERE Access Token.
 
- $ java -jar examples/here-oauth-client-example/target/here-oauth-client-example-*[!javadoc].jar -v -idToken
-
- The -v option is used to print the full valid Id Token to stdout.
+     $ java -jar examples/here-oauth-client-example/target/here-oauth-client-example-*[!javadoc][!sources].jar -idToken
 
+You can optionally add the -v option to print a full valid Id Token to stdout.
```
Comment (sschuberth) [inline README.md]: Also here, formatting with backticks like `-v` would be nice.

**Comment by sschuberth** [comment]
@kenmccracken, looks like you still need to sign our CLA (this is also required for project members, see https://github.com/cla-assistant/cla-assistant/issues/197).

**Comment by sschuberth** [comment]
Finally, the commit message title is a bit too long. Mind putting the part after the comma into the commit message body?

File: README.md (hunk 2 above)
Comment (kenmccracken) [reply README.md]: done.

File: README.md (hunk 1 above)
Comment (kenmccracken) [reply README.md]: done.

**Pull request merged**